mlflow/server/js/src/components/BreadcrumbTitle.js

```diff
@@ -37,7 +38,7 @@ export default class BreadcrumbTitle extends Component {
     }
     const chevron = <i className="fas fa-chevron-right breadcrumb-chevron" key="chevron"/>;
     return (
-      <h1>
+      <h1 style={styles.h1}>
```

Comment:
Why did you use a `style` tag here instead of just putting it in the CSS file? I think we can have `display: inline` on all `h1` elements; I don't see why it would hurt.

Reply:
Changing h1 to `display: inline` I think is kind of unexpected. If you want something adjacent to this, make the parent `display: flex` (preferred, but need to double check in IE11), or use `float`.

Reply:
Ended up making the parent `display: flex`, making the `h1` have an inline style would work too but seemed like a larger change